scripts/aks-auto-start-dry-run.sh

```diff
@@ -24,7 +24,7 @@ jq -c '.[]' <<< $SUBSCRIPTIONS | while read subscription; do
         ENVIRONMENT=$(jq -r '.tags.environment' <<< $cluster)
 
         echo "Test that $APP works in $ENVIRONMENT after $NAME start-up"
-        statuscode=$(curl -s -o /dev/null -w "%{http_code}"  https://$APP.$ENVIRONMENT.platform.hmcts.net)
+        statuscode=$(curl --max-time 30 --retry 5 --retry-delay 15 -s -o /dev/null -w "%{http_code}"  https://$APP.$ENVIRONMENT.platform.hmcts.net)
```

Comment:
do we want max-time? won't that make it so that it only retries once as then it will be over 30s? (from a brief read of the man page anyway)

Reply:
So I seen this about [-m, --max-time](https://curl.se/docs/manpage.html#-m) in the man which made me think it was max time per retry but let me know what you think:

> If you enable retrying the transfer ([--retry](https://curl.se/docs/manpage.html#--retry)) then the maximum time counter is reset each time the transfer is retried. You can use [--retry-max-time](https://curl.se/docs/manpage.html#--retry-max-time) to limit the retry time.

Reply:
ah cool